FusionIIIT/templates/phcModule/appointment.html

```diff
@@ -163,21 +163,21 @@
               $('#inputdate2').attr('min', maxDate);
             });
           </script>
-          <p id="dte"></p>
+          <!-- <p id="dte"></p> -->
         </div>
       </div>
       <div class="field">
         <label>Reason</label>
         <div class="ui fluid input">
           <textarea class="ui textarea" rows="2" id="reason" name="reason"></textarea>
-          <p id="reas"></p>
+          <!-- <p id="reas"></p> -->
```

Comment:
```suggestion
```

Reply:
@akshatnema I've applied the suggested changes. PTAL.